src/controls/RadialProgress.tsx

```diff
@@ -38,8 +45,17 @@ class RadialProgress extends React.Component<IProps, {}> {
   }
 
   public render(): JSX.Element {
-    const { className, data, offset, style, totalRadius } = this.props;
+    const { className, data, offset, style, totalRadius, spin } = this.props;
     const sideLength = (totalRadius + (offset || 0)) * 2;
+
+    if (spin) {
+      return (
+        <svg className={`notifications-spin ${className}`} viewBox={`0 0 ${sideLength} ${sideLength}`} style={style}>
+          {spinData.map(this.renderArc)}
```

Comment:
Also: Why would you have spinData as an array if you don't intend to reuse the existing code? It's static data, you _know_ it's a single element